packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -58,9 +58,7 @@ export class OvsdbClient {
     this._adding.push(adding)
 
     const socket = await this._connect()
-    const index = this._numberOfPortAndInterface
-    ++this._numberOfPortAndInterface
-
+    const index = ++this._numberOfPortAndInterface
```

Comment:
You changed the logic from post-increment, to pre-increment, is it normal?

Reply:
Not a pb, the index will start at 1 instead of 0.
Nothing else changes.